content/modules/redisearch/_index.md

```diff
@@ -7,145 +7,67 @@ categories: ["Modules"]
-enables advanced features, such as language-aware fuzzy matching, fast
-auto-complete, exact phrase matching, numeric filtering and geo-radius
-queries.
-
-You can build compound indexes, indexing multiple fields with text,
-numeric or geo types. RediSearch can be used to index a few types of
-data:
-
-- Indexing new data stored in Redis: If you want to keep all data and
-    index in Redis, this is the best approach. Do this using the ADD
-    command.
-- Index data that is stored in another database: If you already have
-    the data in another database and all you want is a fast, lightweight
-    in-memory index, this can reduce the data redundancy.
-- The RediSearch index only keeps index data and not the full document
-    in Redis. Do this using ADD command with NOSAVE option.
-
-Applications developed with open source version of RediSearch are 100%
-compatible with RediSearch in Redis Enterprise. The primary difference
-between Enterprise RediSearch and the open source version is the
-enterprise version is fully distributed when indexing and processing
-queries, whereas the open source version is not. To understand this
-better, read more below how RediSearch partitions work with RS's
-sharding.
-
-Indexing with RediSearch: RediSearch provides a highly functional index.
-You can build compound indexes that index multiple fields in documents.
-Text, numeric and geospatial data types are supported.
-
-The indexing is done incrementally. As the documents change, you can
-feed the changes to the index. You can keep the copy of documents that
-are indexed to return them as part of the search results. However, this
-is optional. You can limit the index to only contain the index structure
-and results return the unique docID you feed in to help retrieve the
-results from another database that contains the full document.
-
-Search and Query with RediSearch: For full-text searches, developers can
-customize the fields queries and ranking of the search results. When
-querying, developers can use multiple predicates that query both text,
-numeric, and geospatial fields in one query. Sorting can be done by a
-given field as well and can be limited with an offset for simplified
-result-paging support.
-
-RediSearch is language aware and support the following languages for
-stemming. The following list of languages are currently supported with
-RediSearch: "arabic", "danish", "dutch", "english", "finnish", "french",
-"german", "hungarian", "italian", "norwegian", "portuguese", "romanian",
-"russian", "spanish", "swedish", "tamil", "turkish"
-
-The index specifically supports auto-complete engines with specific
-commands that can provide real-time interactive search suggestions.
+The RediSearch 2.x module is a source-available project that lets you build powerful searches for Redis databases.
+When combined with Redis Enterprise Software (RS), you can use the same RediSearch protocols and commands
```

Comment:
What do you mean by "the same RediSearch protocols"? You mean the RediSearch 1.x protocols?